cmake/SettingsRegistry.cmake

```diff
@@ -118,6 +118,11 @@ endfunction()
 #  The generated file contains the file to the each dependent targets
 #  This can be used for example to determine which list of gems to load with an application
 function(ly_delayed_generate_settings_registry)
+
+    if(LY_MONOLITHIC_GAME) # No need to generate setregs for monolithic builds
+        return()
+    endif()
```

Comment:
Actually we shouldn't just return here.
We should also clear out the `LY_LOAD_DEPENDENCIES` global property as we do at the end of these function normally

`set_property(GLOBAL PROPERTY LY_DELAYED_LOAD_DEPENDENCIES)`

Reply:
sure, although that variable is not used after right? will clear it

Reply:
We could clear it, but we don't have to since those variables "naming" concatenation is based on `LY_DELAYED_LOAD_DEPENDENCIES` global property (prefix, target) pair.
Since we cleared out the `LY_DELAYED_LOAD_DEPENDENCIES`, any further script logic couldn't access the `$LY_DELAYED_LOAD_"${prefix_taget}"` global property.
Plus it is not like CMake is persisting the properties to disk, so disk space isn't a concern we have to deal with.